bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ_CASE/tZ_b_LH_conf/proc_card.dat

```diff
@@ -0,0 +1,9 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model STP_UFO_freeWidth
+define bb = b b~
+define tt = t t~
+generate p p > t23 > j bb tt z / h
+add process p p > t23~ > j bb tt z / h
```

Comment:
Hi, I am not very aware of this model.
t23 goes to t z? And what becomes j b?

Reply:
Hi Sihyun,
This is basically just an adaptation of these cards: https://github.com/wpmccormack/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeToTZ/tZt_NW/tZ_t_LH_M700.  When I look at the lhe file for this process, I see e.g.:
<event>
13      2 +1.8079137e-02 5.88367800e+01 7.81860800e-03 1.22107000e-01
        1 -1    0    0  501    0 +0.0000000000e+00 +0.0000000000e+00 +2.1080922399e+03 2.1080922399e+03 5.0400000000e-03 0.0000e+00 -1.0000e+00
       21 -1    0    0  502  503 +0.0000000000e+00 +0.0000000000e+00 -1.0677618059e+02 1.0677618059e+02 0.0000000000e+00 0.0000e+00 1.0000e+00
 -8000001  2    1    2    0  503 +9.3935538870e+00 +5.3269099979e+01 +1.3275558431e+03 1.5017765418e+03 7.0000173511e+02 0.0000e+00 0.0000e+00
        2  1    1    2  501    0 -7.9086053433e+00 -5.2719893388e+01 +6.9207485872e+02 6.9412501991e+02 2.5500000000e-03 0.0000e+00 -1.0000e+00
        5  1    1    2  502    0 -1.4849485437e+00 -5.4920659035e-01 -1.8314642536e+01 1.8966858781e+01 4.6700000000e+00 0.0000e+00 1.0000e+00
       -6  2    3    3    0  503 +9.0055722657e+01 -2.3477595110e+02 +1.0132552686e+03 1.0578834954e+03 1.7088437638e+02 0.0000e+00 0.0000e+00
       -5  1    6    6    0  503 +9.0689383231e+01 -5.1817698858e+01 +3.3640708190e+02 3.5227996224e+02 4.6700000000e+00 0.0000e+00 1.0000e+00
      -24  2    6    6    0    0 -6.3366057415e-01 -1.8295825225e+02 +6.7684818669e+02 7.0560353315e+02 7.9238592675e+01 0.0000e+00 0.0000e+00
        3  1    8    8  504    0 +1.2190152472e+01 -1.1048576041e+02 +2.6723564169e+02 2.8943151428e+02 1.0100000000e-01 0.0000e+00 -1.0000e+00
       -2  1    8    8    0  504 -1.2823813046e+01 -7.2472491839e+01 +4.0961254500e+02 4.1617201886e+02 2.5500000000e-03 0.0000e+00 1.0000e+00
       23  2    3    3    0    0 -8.0662168770e+01 +2.8804505108e+02 +3.1430057454e+02 4.4389304641e+02 9.3700846149e+01 0.0000e+00 0.0000e+00
       12  1   11   11    0    0 -1.4615239670e+01 +1.5708851577e+02 +1.0971979994e+02 1.9216878392e+02 0.0000000000e+00 0.0000e+00 -1.0000e+00
      -12  1   11   11    0    0 -6.6046929100e+01 +1.3095653531e+02 +2.0458077460e+02 2.5172426248e+02 0.0000000000e+00 0.0000e+00 1.0000e+00

Basically, there seems to be an ISR jet, then Tprime + B production.  The Tprime then decays to top + Z.

Reply:
note that with the above syntax lorentz and gauge variance tests fail in mg5. did you ever compare the kinematics etc to p p > t23 j bb , t23 > tt Z ?